### DISCUSSION TITLE
Cyrillic font doesn't show up in the browser

### DISCUSSION THREAD

**Discussion by aptGrammatist** [ruffle-rs/ruffle#15540] [Q&A]
The title says it all

Screenshot in the desktop version
![Знімок екрана 2024-03-12 о 14 48 16](https://github.com/ruffle-rs/ruffle/assets/157318917/cc94625b-6b02-4b21-8109-544d31a0533c)

Screenshot in the demo
![Знімок екрана 2024-03-12 о 14 48 46](https://github.com/ruffle-rs/ruffle/assets/157318917/5e533773-2cfb-48c0-9898-14d3b4ac181f)

**Comment by aptGrammatist** [comment]
> The title says it all
> 
> Screenshot in the desktop version ![Знімок екрана 2024-03-12 о 14 48 16](https://private-user-images.githubusercontent.com/157318917/312080789-cc94625b-6b02-4b21-8109-544d31a0533c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAyNDgzMTYsIm5iZiI6MTcxMDI0ODAxNiwicGF0aCI6Ii8xNTczMTg5MTcvMzEyMDgwNzg5LWNjOTQ2MjViLTZiMDItNGIyMS04MTA5LTU0NGQzMWEwNTMzYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxMlQxMjUzMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZTE4NWRiZWY2Mzg1ZGRmNGJiOWVjYjZiZDBiYTY0MTJlZGVjZWY3NWFkMTJhYTQ2MzMyODU4YTNmZjFhMzAwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.AYyjw0woZxA98xXHS5MrV9P5rQ-SvQNfuBaefEX-TXU)
> 
> Screenshot in the demo ![Знімок екрана 2024-03-12 о 14 48 46](https://private-user-images.githubusercontent.com/157318917/312080906-5e533773-2cfb-48c0-9898-14d3b4ac181f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAyNDgzMTYsIm5iZiI6MTcxMDI0ODAxNiwicGF0aCI6Ii8xNTczMTg5MTcvMzEyMDgwOTA2LTVlNTMzNzczLTJjZmItNDhjMC05ODk4LTE0ZDNiNGFjMTgxZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxMlQxMjUzMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYTNiZWUxMmJiODY4NTgyOTA1YWVkZWNiMWY1M2I3ZTM2MWQ3ZGZiMzBkZmVhODRlNWFhNDdjNGRmMGQwYWM5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.khtpjQyNPrVP4CHvP41420Gg5CjCQqMkl2C-P58Q0Kc)

Link to the file: https://file.garden/ZU-xUYjV4SO9KYcO/hsua/%5BS%5D%D1%82%D0%BE%D1%80%D1%96%D0%BD%D0%BA%D0%B8/00137.swf

**Reply by Lord-McSweeney** [reply]
You need to include the Cyrillic font in a font SWF. See #13604, specifically the "How to make a font SWF" section.
